style.css

```diff
@@ -0,0 +1,150 @@
+:root {
+  --ecomm-primary-color: rgba(17, 144, 230, 0.733);
+  --secondary-color: rgb(255, 255, 255);
+  --head-text-color: rgb(116, 115, 115);
+  --light-grey-color: rgb(238, 235, 235);
+  --red-color: red;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.container .header {
+  background-color: var(--ecomm-primary-color);
+  height: 4.5rem;
+}
+
+.container .header .logo {
+}
+
+.search-container {
+  border: 1px solid var(--light-grey-color);
+  border-radius: 10px;
+}
+
+.header .search-container #search {
+  color: black;
+}
+
+.container .action-icon .login {
+  color: var(--head-text-color);
+  background: none;
+  padding: 0;
+}
+
+.container .header .action-icon .icon i,
+.action-icon .login i {
+  color: var(--secondary-color);
+}
+
+.container .header .action-icon .icon span {
+  color: var(--secondary-color);
+  background-color: var(--red-color);
+}
+
+.hero-banner-container {
+  position: relative;
+  height: 50rem;
+  overflow: hidden;
+  background: url(https://dispatch.barnesandnoble.com/content/dam/ccr/homepage/daily/2022/02/01/Gateway_Billboard-C_OMP.jpg)
+    no-repeat center;
+  background-size: cover;
+}
+
+.hero-image {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  overflow: hidden;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.hero-banner-container .btn {
+  background-color: aqua;
+}
+
+.new-release {
+  /* padding: 2rem 6rem; */
+  /* border: 1px solid black; */
+}
+
```

Comment:
Remove unnecessary comments and classes whereever possible.
rest everything looks good.

Reply:
removed